cleanlab_studio/studio/studio.py

```diff
@@ -457,6 +458,50 @@ def poll_cleanset_status(self, cleanset_id: str, timeout: Optional[int] = None)
         except (TimeoutError, CleansetError):
             return False
 
+    def enrich_data(
```

Comment:
could you look at the notion doc more carefully regarding where all the functions should live? remember this should not be user-facing

Reply:
the user-facing `enrich_data` method should be an instance method. I forgot it requires a `Studio` client object. Can you just make that client object a required arg of the `enrich_data()` method too?

This is not good long-term, but easiest way for now to keep this method deep in the utils folder vs. being user-facing

Reply:
I did not want to create a separate file in utils folder, instead added it to `cleanlab_studio.internal.util.py`